test/libyul/yulStackShuffling/swap17.stack

```diff
@@ -0,0 +1,51 @@
+[ junk v00 v01 v02 v03 v04 v05 v06 v07 v08 v09 v10 v11 v12 v13 v14 junk deep ]
+[ deep v14 v13 v12 v11 v10 v09 v08 v07 v06 v05 v04 v03 v02 v01 v00 ]
+// ====
+// maximumStackDepth: 256
+// ----
+// [ junk v00 v01 v02 v03 v04 v05 v06 v07 v08 v09 v10 v11 v12 v13 v14 junk deep ]
+// SWAP17
```

Comment:
Note: this only tests that the stack shuffler doesn't produce shuffling code with one more operation to avoid a stack-too-deep here, since in this case this would be possible (see ``pop_to_avoid_swap17.stack``). In cases in which a stack-too-deep cannot be avoided, the stack shuffler will still produce ``SWAP17`` to trigger the stack-too-memory logic (regardless of the maximum stack depth), so e.g. for a full reversal of 17 slots, the expectations won't differ between maximum stack depth settings.